### PR TITLE
#57 - Moved tests over to use fastify inject

### PR DESCRIPTION
**Pull request by Svjard**
#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Svjard** [comment]
@mcollina Very nice! Yes that would at least clean some of this up and simplify it. I'll keep an eye out for the next release of fastify to incorporate it and update accordingly.

**Comment by mcollina** [comment]
it's already in!

**Comment by Svjard** [comment]
Updated to use @mcollina new cookie feature.

File: test/cookie.test.js [hunk 1]
```diff
@@ -23,21 +21,17 @@ test('cookies get set correctly', (t) => {
     if (err) tap.error(err)
```
Comment (jsumners) [inline test/cookie.test.js]: `fastify.inject` does not require that the server be running and listening on a real port. So all of these `fastify.listen(0, (err) => {})` blocks can be unwrapped.

**Pull request merged**